src/plugins/cordova-plugin-network-information/sim-host.js

```diff
@@ -30,6 +37,18 @@ function initialize() {
     connectionTypeList.addEventListener('change', function () {
         network.updateNetworkType(connectionTypeList.value);
     });
+
+    registerTelemetryEvents();
+}
+
+function registerTelemetryEvents() {
+    var connectionList = document.getElementById('connection-list');
+
+    connectionList.onchange = function () {
+        var option = connectionList.options[connectionList.selectedIndex];
+
+        telemetry.sendUITelemetry(Object.assign({}, baseProps, { control: 'connection-list', value: option.value }));
+    };
```

Comment:
Of course, the questions are totally valid :) Here's what we think:
We believe this will help us understand how the UI is being used, what are the popular plugins, how frequently do people change the simulation values, etc. It will help us make decisions about future investments, how we prioritize plugin simulation work, etc